requirements.txt

```diff
@@ -3,6 +3,7 @@ boto3
 bottle
 cffi
 click
+colorama
```

Comment:
Do we still need this?

Reply:
Yes, there are features in click that this might prove useful in the future.